tests/src/Traits/RdfEntityTrait.php

```diff
@@ -14,26 +14,29 @@
    *
    * If multiple RDF entities have the same name the first one will be returned.
    *
-   * @param string $title
-   *   The RDF entity title.
+   * @param string $label
+   *   The RDF entity label.
    * @param string $type
-   *   The RDF entity type.
+   *   Optional RDF entity type.
    *
    * @return \Drupal\rdf_entity\Entity\Rdf
    *   The RDF entity.
    *
    * @throws \InvalidArgumentException
    *   Thrown when an RDF entity with the given name and type does not exist.
    */
-  protected function getRdfEntityByLabel($title, $type) {
+  protected function getRdfEntityByLabel($label, $type = NULL) {
```

Comment:
This is not true. Rdf entities are supposed to have unique titles per bundle and not across site. And of course this is not the case, but for example, a license can have the same name as a collection and a release with it's solution (at least for now).